specifications/Staking Lockup.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Staking lockup to incentivize illiquidity
+author: @vtleonardo, @nelsonhp
+discussion: https://github.com/alicenet/alicenet/pull/340
+team: Backend
+category: Development
+status: Review
+related: -
+created: 2022-11-14
+---
+
+## Introduction
+
+### Summary
+
+This specification proposes a series of mechanisms to incentivize the illiquidity of ALCA token at the beginning of the AliceNet lifetime. The main component of this mechanism will be a Lockup smart contract that will allow users to lock their public staking positions in exchange for gaining extra rewards.
+
+### Context
+
+Once we release the Alicenet economics system, users will be able to get ALCA by migrating the old token (MadToken) directly into the ALCA smart contract. In addition to this, we also foresee the creation of AMM pools with ALCA in popular markets (UNISWAP V3, Balancer, etc), where users will be able to buy ALCA directly using different currencies.
+
+With ALCA tokens in hand, users will be able to stake into the Public Staking contract, and in the near future, they will be able to participate in the governance system to decide the future of the protocol.
```

Comment:
I think, this paragraph should be incorporated into the first to make it more clear to the reader that they they will receive profits iff their alca is staked in the publioc staking contract, and they will receive bonus alca rewards iff they have their staked postion locked.